test/test-v8debugapi.ts

```diff
@@ -1056,10 +1102,12 @@ describe('v8debugapi', function() {
               'Max data size reached'));
             assert(((fooVal as any).status as any).isError);
 
-            api.clear(bp);
             config.capture.maxDataSize = oldMaxData;
```

Comment:
The same concern about moving these lines inside the callback of `api.clear` as described above.